crypto/impl_dispatch_test.cc

```diff
@@ -140,6 +140,11 @@ class ImplDispatchTest : public ::testing::Test {
   bool is_assembler_too_old = false;
   bool is_assembler_too_old_avx512 = false;
   bool ifma_avx512 = false;
+  bool is_bn_mulx4x_mont_gather5 = false;
```

Comment:
I'm not sure that we need these five values defined here, we only define capabilities here?